docs/index.rst

```diff
@@ -80,17 +75,13 @@ A sample python test case with test case tokens is shown below:
         @automated: false
         """
 
-In the above example, as you may guess - `feature`, `setup`, `steps`, `assert`,
-`bz`, `automated` are all tokens.
-
-.. _how_it_works:
+In the above example, as you may guess - ``feature``, ``setup``, ``steps``,
+``assert``, ``bz``, ``automated`` are all tokens.
 
 How it works?
 `````````````
 
-To understand how Testimony works, let's look at the ``help`` command:
-
-.. code-block:: bash
+To understand how Testimony works, let's look at the ``help`` command::
 
```

Comment:
After submitting my PR, I realized that I should have used `console` attribute.  Can you please put back the `code-block` statements replacing bash with `console`?

`.. code-block:: console`